ste/folderCreationTracker.go

```diff
@@ -126,8 +125,7 @@ func (f *jpptFolderTracker) ShouldSetProperties(folder string, overwrite common.
 
 		var created bool
 		if fNode, ok := f.contents.GetDirNode(folder); ok {
-			created = f.plan.Transfer(fNode.TransferIndex).TransferStatus() == common.ETransferStatus.FolderCreated() ||
-				f.plan.Transfer(fNode.TransferIndex).TransferStatus() == common.ETransferStatus.Success()
+			created = f.plan.Transfer(fNode.TransferIndex).TransferStatus() == common.ETransferStatus.FolderCreated()
```

Comment:
Earlier when Dhanashree was making this change we observed that the directory entry shows created for some time but later the entry becomes success and if we do not check for the success status here more create directories will follow through.

Reply:
yep, Adele is working on a more correct fix, which is why I decided to close this.